### PR TITLE
Refactor the Calculator component

### PR DESCRIPTION
**Pull request by Alfredbis29**
**In this milestone of my project I**

- Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.
- After the refactor is done, I test the app in the browser and It keeps the same functionality.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,71 +1,57 @@
...
-      obj: {
-        total: null,
-        next: null,
-        operation: null,
-      },
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Calculator() {
+  const [calcObj, setCalcObj] = useState({
```
Comment (elmar8287) [inline src/components/Calculator.js]: - [x] Please, consider using arrow function syntax 👍 as it is shown in an article in project requirements - I know you can do it easily 🥇 

```
const Calculator = () => {
....
}

export default Calculator;
```

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Alfredbis29.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues
- Informative README file

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- Please, try to fix the broken link to MIT.md file 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Alfredbis29) [reply src/components/Calculator.js]: Hello @elmar8287  thanks so much for the professional code review, 
I made what was required,  using arrow function syntax.
You can now check

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Alfredbis29 , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfredbis29** [comment]
Hey There @ShoiraTa, Thanks so much for your approval, you are amazing!

**Pull request merged**